ClickHouse.Client/ClickHouseUriBuilder.cs

```diff
@@ -25,29 +25,44 @@ public ClickHouseUriBuilder(Uri baseUri)
 
     public string SessionId { get; set; }
 
+    public string QueryId { get; set; }
+
     public static string DefaultFormat => "RowBinaryWithNamesAndTypes";
 
-    public IDictionary<string, object> CustomParameters { get; set; }
+    public IDictionary<string, object> ConnectionQueryStringParameters { get; set; }
```

Comment:
This class doesn't need to know about existence of Command or ConnectionString. The custom parameters should be fed from outside - overriding the values as needed

Reply:
Good point. However, considering that this class is internal, I believe this implementation is easier to maintain.

Reply:
Having N properties instead of 1 is definitely not easier to maintain. Merging the dictionary can well be done at builder population stage rather than at URI build time

Reply:
Merging this into a single dictionary would complicate unit tests. To properly test the override of command-level settings over connection-level ones, I'd need to create a connection with a command and mock an `HttpClient`. Since there is no Moq reference yet, what do you think is the most suitable approach here?